Aplib.Core/ILoggable.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections.Generic;
+
+namespace Aplib.Core
+{
+    /// <summary>
+    /// An interface that allows defining the structure loggable objects through a tree.
+    /// </summary>
```

Comment:
I think it would be nice to have a bit more in-depth documentation here, explaining the tree structure and how to use this. But if you don't have time this weekend, I understand, then just leave it like this, but do add an ‘of’ between “the structure” and “loggable objects”.